Issue #3415660 by govind.maloo: Arrows supporting Previous and Next buttons on webforms are not rendering as expected

govindmaloo opened the pull request:
https://www.drupal.org/project/civictheme/issues/3415660

## Checklist before requesting a review

- [x] I have formatted the subject to include ticket number as `[CS-123] Verb in past tense with dot at the end.`
- [x] I have added a link to the issue tracker
- [ ] I have provided information in `Changed` section about WHY something was done if this was not a normal implementation
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run new and existing relevant tests locally with my changes, and they passed
- [ ] I have provided screenshots, where applicable

## Changed

1.

## Screenshots

AlexSkrypnyk commented:
@govindmaloo 
Did you try to use the `raw` filter: 
```twig
value: attributes.value|raw
```

AlexSkrypnyk commented on web/themes/contrib/civictheme/includes/form.inc:
```diff
@@ -45,6 +45,13 @@ function civictheme_preprocess_input(array &$variables): void {
+  // Fix htmlentity in submit button text.
+  $element = $variables['element'];
+
+  if (isset($element['#type']) && $element['#type'] == 'submit' && !empty($variables['attributes']['value'])) {
```
what if this is type `reset` or just a button? the solution should take care of all possible cases

govindmaloo replied:
I have tried raw filter but didn't work.

And it should work for all button. I can review if you think this solution can work.

AlexSkrypnyk commented on web/themes/contrib/civictheme/templates/form/input--submit.html.twig:
```diff
@@ -17,7 +18,7 @@
 {% set children %}
   {% include "@atoms/button/button.twig" with {
     type: type,
-    text: attributes.value,
```
does `attributes.value` contains HTML-converted text? can you try using `raw` filter?

govindmaloo commented:
@AlexSkrypnyk  I have updated this branch on top of the latest develop and also added this PR to fix CI build
https://github.com/civictheme/monorepo-drupal/pull/1211

AlexSkrypnyk commented on web/themes/contrib/civictheme/templates/form/container.html.twig:
```diff
@@ -7,6 +7,7 @@
 {% set classes = [
     'container',
     has_parent ? 'js-form-wrapper form-wrapper',
+    'ct-contianer',
```
there is no class `ct-container`. Also, this is misspelt

AlexSkrypnyk commented:
@govindmaloo
I have updated the head branch to point to `develop` and this branch now contains unrelated changes.

Could you please remove any unrelated changes and only leave what is necessary for this issue.

Thanks
